app/test_main.py

```diff
@@ -1,3 +1,63 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, error",
+    [
+        ([2], 8, TypeError),
+        (10, [1], TypeError),
+        ("hello", "13", TypeError)
+    ]
+)
+def test_raise_error_if_value_types_are_incorrect(
+        cat_age: int | list | str,
+        dog_age: int | list | str,
+        error: type[Exception]
+) -> None:
+    with pytest.raises(error):
+        get_human_age(cat_age, dog_age)
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_result",
```

Comment:
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/103201010/22a4b874-399b-46b6-be34-1cda682f64ff)
what about zeros and big nums ?